wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialise in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
+
+## osu! Development server
+
+The **osu! Development** server — also known as **osu!dev** or just **osu!** — is the game's official Discord server. It is meant to be the main place to view and discuss osu!'s ongoing developments, a place to contribute to the codebase, and a place where a plethora of community projects are organised in!
+
+**This is the only server officially affiliated with the game.** To keep up with osu!'s active development, it is strongly suggested to join the server.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu!](https://discord.gg/ppy) | [peppy](https://osu.ppy.sh/users/2) | The **osu!** Discord server, previously known as **osu!dev**, is osu!'s official Discord server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the [osu! team](/wiki/People/osu!_team). |
```

Comment:
let's put country flags on the owners then cuz that normally happens throughout the wiki